### ISSUE TITLE
CBS episode output template includes series title

### ISSUE THREAD

**Issue by kucksdorfs** [ytdl-org/youtube-dl#23922]
## Checklist

- [X] I'm asking a question
- [X] I've looked through the README and FAQ for similar questions
- [X] I've searched the bugtracker for similar questions including closed ones

## Question

I am not sure if this is a bug, or if this is more appropriate as a question. I figured a question was safer than saying this was a bug. I started playing with using the templates when using youtube-dl and noticed that when I use CBS urls for the template '%(title)s' it creates a file with the series name and the name of the episode.
For example:

`youtube-dl --get-filename -o '%(title)s' https://www.cbs.com/shows/blue_bloods/video/1jEOtccsxwFtS
tuH86_Gzsk_6i_9KBTz/blue-bloods-reckless/`

gives the output "Blue Bloods - Reckless", but when I do the same with NBC or ABC URLS

`
youtube-dl --get-filename -o '%(title)s' https://abc.com/shows/modern-family/episode-guide/season-11/12-dead-on-a-rival
youtube-dl --get-filename -o '%(title)s' https://www.nbc.com/manifest/video/black-box/4103972
`

I get the following:

Dead on A Rival
Black Box

There appears to be a minor discrepancy in the %(title)s format.
I would expect title would return the episode title only and fulltitle to do the series title and episode title.

On a side note, it looks like fulltitle doesn't do series name and episode name (it looks like in all three urls provided %(title)s is the same as %(fulltitle)s). 

tl;dr; The %(title) shows "{seriesname} - {episodename}" when I think it should just be the episode name. Should it be that way?

**Comment by dstftw** [comment]
`title` is provided as is. Actual value depends on a particular provider.
